processor/lib/partitionScanner.ts

```diff
@@ -207,7 +207,7 @@ export class PartitionScanner {
           } else if (lease) {
             this._leaseOwnedByOthers.set(lease.partitionId, lease);
           }
-          return;
+          return Promise.resolve();
```

Comment:
Is the return line even needed?

Reply:
Yes, otherwise tslint reports an error
```
> tslint -p . -c tslint.json

lib/partitionScanner.ts:198:17 - error TS7030: Not all code paths return a value.

198         }).then((acquired) => {
                    ~~~~~~~~~~~~~~~


Found 1 error.
```